src/D2L.Bmx/Okta/OktaApi.cs

```diff
@@ -88,9 +88,10 @@ internal class OktaApi : IOktaApi {
 		}
 
 		string org = _organization ?? "unknown";
-		throw new BmxException(
-			$"Okta authentication for user '{username}' in org '{org}'"
-				+ "failed. Check if org, user, and password is correct" );
+		throw new BmxException( $"""
+			Okta authentication for user '{username}' in org '{org}' failed.
+			Check if org, user, and password is correct.
+			""" );
```

Comment:
This one was missing a space before "failed", and the usage of period was inconsistent:
![image](https://github.com/user-attachments/assets/693fd0c6-0d9c-4398-a4ff-404383f89cde)

Easier (and clearer, I think) to use a raw string literal with multiple lines instead.